gensim/models/lda_worker.py

```diff
@@ -113,26 +226,32 @@ def reset(self, state):
 
     @Pyro4.oneway
     def exit(self):
+        """Terminate the worker."""
         logger.info("terminating worker #%i", self.myid)
         os._exit(0)
 
 
 def main():
+    """Set up argument parser,logger and launches pyro daemon."""
     parser = argparse.ArgumentParser(description=__doc__)
-    parser.add_argument("--host", help="Nameserver hostname (default: %(default)s)", default=None)
-    parser.add_argument("--port", help="Nameserver port (default: %(default)s)", default=None, type=int)
+    parser.add_argument("--host", help="Nameserver hostname \
+        (default:%(default)s)", default=None)
+    parser.add_argument("--port", help="Nameserver port \
+        (default: %(default)s)", default=None, type=int)
     parser.add_argument(
-        "--no-broadcast", help="Disable broadcast (default: %(default)s)", action='store_const',
-        default=True, const=False
-    )
-    parser.add_argument("--hmac", help="Nameserver hmac key (default: %(default)s)", default=None)
+        "--no-broadcast", help="Disable broadcast \
```

Comment:
why reformatting? we using `120 characters` limit

Reply:
oh, I ran flake8 and it was giving error for lines above 79 chars. Anyways ,I will change it then.

Reply:
you should use our flake config: `tox -e flake8`